src/test/java/gov/hhs/cms/bluebutton/fhirstress/utils/BenefitIdMgrTest.java

```diff
@@ -0,0 +1,41 @@
+package gov.hhs.cms.bluebutton.fhirstress.utils;
+
+import org.junit.*;
+import static org.junit.Assert.*;
+
+public class BenefitIdMgrTest {
```

Comment:
Great! Glad to see tests. I'd add one for thread safety, though (assuming that I'm right and it ought to be thread safe).

Reply:
I added synchronized keyword to methods, I think that should fix it based on what I have read on the interwebs.